docs/machine-learning/how-to-guides/train-machine-learning-model-ml-net.md

```diff
@@ -0,0 +1,179 @@
+
+By using the [`Concatenate`](xref:Microsoft.ML.TransformExtensionsCatalog.Concatenate*) method during pre-processing to create a new column called `Features`, there is no need to specify the feature column name in the parameters of the algorithm since it already exists in the pre-processed `IDataView`. The label column is `CurrentPrice`, but since the [`ColumnName`](xref:Microsoft.ML.Data.ColumnNameAttribute.#ctor*) attribute is used in the data model, ML.NET renames the `CurrentPrice` column to `Label` which removes the need to provide the `labelColumnName` parameter to the machine learning algorithm estimator. 
+
+If you don't want to use the default column names, pass in the names of the feature and label columns as parameters when defining the machine learning algorithm estimator as demonstrated by the subsequent snippet:
+
+```csharp
+var UserDefinedColumnSdcaEstimator = mlContext.Regression.Trainers.Sdca(labelColumnName: "MyLabelColumnName", featureColumnName: "MyFeatureColumnName");
+```
+
+## Train the machine learning model
+
+Once the data is pre-processed, use the [`Fit`](xref:Microsoft.ML.Trainers.TrainerEstimatorBase`2.Fit*) method to train the machine learning model with the [`StochasticDualCoordinateAscent`](xref:Microsoft.ML.Trainers.SdcaRegressionTrainer) regression algorithm.
+
+```csharp
+// Define StochasticDualCoodrinateAscent regression algorithm estimator
+var sdcaEstimator = mlContext.Regression.Trainers.Sdca();
+
+// Build machine learning model
+var trainedModel = sdcaEstimator.Fit(transformedTrainingData);
+```
+
+## Extract model parameters
+
+After the model has been trained, extract the learned [`ModelParameters`](xref:Microsoft.ML.Trainers.ModelParametersBase`1) for inspection or re-training. The [`LinearRegressionModelParameters`](xref:Microsoft.ML.Trainers.LinearRegressionModelParameters) provide the bias and learned coefficients or weights of the trained model. 
+
+```csharp
+var trainedModelParameters = trainedModel.Model as LinearRegressionModelParameters;
+```
+
+> [!NOTE]
+> Other models have parameters that are specific to their tasks. For example, K-Means has centroids and the [`KMeansModelParameters`](xref:Microsoft.ML.Trainers.KMeansModelParameters) contains a property that stores these learned centroids. To learn more, visit the [`Microsoft.ML.Trainers` API Documentation](xref:Microsoft.ML.Trainers) and look for classes that contain `ModelParameters` in their name. 
+
+## Evaluate model quality
+
+To help choose the best performing model, it is essential to evaluate its performance on unseen data. Use the [`Evaluate`](xref:Microsoft.ML.RegressionCatalog.Evaluate*) method, to measure calculate various metrics for the trained model.
```

Comment:
>unseen [](start = 89, length = 6)

test data (because they have to have a label.)
unseen data is mostly used for prediction.